xmake/rules/c++/modules/modules_support/clang.lua

```diff
@@ -489,7 +489,11 @@ function get_builtinmodulemapflag(target)
     if builtinmodulemapflag == nil then
         local compinst = target:compiler("cxx")
         if compinst:has_flags("-fbuiltin-module-map", "cxxflags", {flagskey = "clang_builtin_module_map"}) then
-            builtinmodulemapflag = "-fbuiltin-module-map"
+            if target:is_plat("windows", "mingw") then
```

Comment:
windows plat will use msvc.

Reply:
for clang-cl? did you test it?

Reply:
Tested in windows msys2(mingw-w64) environment and it works fine. msvc and clang-cl are not part of the mingw environment.

Reply:
so we should just use ` if target:is_plat("mingw") then`

Reply:
No. linux also has a mingw backend.

Reply:
but you should not add the windows platform, which is always used for msvc.

Reply:
I don't understand how we get inside that `if`, shouldn't `compinst:has_flags("-fbuiltin-module-map", "cxxflags")` return false for mingw?

Reply:
The flag can be supported but but the modulemap file can be missing, and it seems clang on mingw doesn't distribute it